src/Npgsql/NpgsqlCommand.cs

```diff
@@ -550,19 +550,15 @@ void DeriveParametersForQuery()
         /// Creates a server-side prepared statement on the PostgreSQL server.
         /// This will make repeated future executions of this command much faster.
         /// </summary>
-        public Task PrepareAsync() => PrepareAsync(CancellationToken.None);
-
-        /// <summary>
-        /// Creates a server-side prepared statement on the PostgreSQL server.
-        /// This will make repeated future executions of this command much faster.
-        /// </summary>
+        /// <param name="cancellationToken">The token to monitor for cancellation requests. The default value is <see cref="CancellationToken.None"/>.</param>
 #if !NET461 && !NETSTANDARD2_0
-        public override Task PrepareAsync(CancellationToken cancellationToken)
+        public override Task PrepareAsync(CancellationToken cancellationToken = default)
 #else
         public Task PrepareAsync(CancellationToken cancellationToken)
```

Comment:
You should add `default` here too because of removed parameterless `PrepareAsync`.

Reply:
Done